ilc/config/test.json5

```diff
@@ -0,0 +1,24 @@
+{
+    productionMode: false,
+    registry: {
```

Comment:
Could you wrap all these keys except `productionMode: false` with `e2eTests` key?
So you'll have `e2eTests.registry.port`